### PR TITLE
Fixes for the PR comments in #35

### PR DESCRIPTION
**Pull request by maelvls**
This is a follow-up PR to #35. In #35, I had to merge the PR as soon as possible and was not able to push the changes related to Richard's PR comments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -271,7 +271,7 @@ service that enables you to simplify, automate, and customize the
 deployment, management, and security of private certificate authorities
 (CA).
 
-[Google Certificate Authority Service]: https://cloud.google.com/certificate-authority-service/
+[google certificate authority service]: https://cloud.google.com/certificate-authority-service/
```
Comment (wallrj) [inline README.md:274]: Why lower case? The title of that page is "Certificate Authority Service" and "Google" should always be capitalized.

**Review by wallrj — APPROVED** [review]
Thanks for all the extra detail @maelvls 

If I was going to nitpick, I'd probably ask for even more detail on the pricing mechanism and ubbagent, but that can be in a followup.

I see there's a note about pricing in https://console.cloud.google.com/marketplace/details/jetstack-public/jetstack-secure-for-cert-manager?project=jetstack-public&authuser=0, and I wondered if that should also be added to the README file in this repo, for the benefit of users who take the CLI installation route.

Address or answer these points and unhold if you'd like to get this merged. 

Thanks.

/lgtm
/hold

File: README.md (hunk 1 above)
Comment (maelvls) [reply README.md]: Oops, you are right, it should be upper-cased! I think my linter (Prettier) did this 😇

**Review by wallrj — APPROVED** [review]
/lgtm
/unhold

**Comment by maelvls** [comment]
> I'd probably ask for even more detail on the pricing mechanism and ubbagent, but that can be in a followup. I see there's a note about pricing in https://console.cloud.google.com/marketplace/details/jetstack-public/jetstack-secure-for-cert-manager?project=jetstack-public&authuser=0, and I wondered if that should also be added to the README file in this repo, for the benefit of users who take the CLI installation route.

Good point, I'll open another PR with better explanations around that 👍

**Comment by maelvls** [comment]
> I wondered if that should also be added to the README file in this repo, for the benefit of users who take the CLI installation route.

The CLI flow still includes browsing to the main application page (which contains the $0.97/hour price) in order to get to the "billing secret" under the "Install using the CLI" tag after clicking "Configure".

I'll add a note about the pricing in the README though. I think it is good to make that clear.

**Pull request merged**